blosc/frame.c

```diff
@@ -2393,6 +2393,31 @@ void* frame_update_chunk(blosc2_frame_s* frame, int nchunk, void* chunk, blosc2_
       return NULL;
     }
   }
+  int32_t cbytes_old;
+  int32_t old_offset;
+  if (!frame->sframe) {
+    // See how big would be the space
+    old_offset = offsets[nchunk];
+    bool needs_free;
+    uint8_t *chunk_old;
+    int err = blosc2_schunk_get_chunk(schunk, nchunk, &chunk_old, &needs_free);
+    if (err < 0) {
+      BLOSC_TRACE_ERROR("%d chunk con not obtenined from schunk.", nchunk);
+    }
+
+    if (chunk_old == 0) {
```

Comment:
It is better to use NULL with pointers.

Reply:
Fixed in 9c3bf17a428af246f12b85fe2816bc0dd5177a98.
The `test_update_chunk` runs this in the case of a contiguous frame.